README.md

```diff
@@ -12,7 +12,11 @@ Contributions are welcome!
 
 Clone locally with:
 
-    git clone https://github.com/OriginProtocol/docs.git origin-docs && origin-docs
+    git clone https://github.com/OriginProtocol/docs.git origin-docs && cd origin-docs
+
+Make sure you have Ruby installed or install (e.g. with Homebrew on Mac):
+
+    brew install ruby
```

Comment:
Need ruby > 2.3.1 and I would generally recommend `rvm`, but people may be fine with `brew`